src/style.css

```
@@ -0,0 +1,84 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+* {
+  box-sizing: border-box;
+
+  padding: 0%;
+  margin: 0%;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 16px;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  padding-top: 3px;
+  padding-left: 50px;
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+  height: 60px;
+  margin-right: 50px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  display: inline-block;
+  margin-right: 20px;
+}
```

Keep the associated code together

```suggestion
.nav__item {
  display: inline-block;
  margin-right: 20px;
}

.nav__item:last-child {
  margin-right: 0;
}
```